src/components/ErrorBoundary/ErrorBoundaryIcon.tsx

```diff
@@ -0,0 +1,46 @@
+const ErrorIcon = () => (
```

Comment:
Do we actually need this SVG as a component or could it be used as an image? It doesn't seem to change overtime or to be interactive